Configuration/AlCa/python/autoCond.py

```diff
@@ -86,7 +86,7 @@
     # GlobalTag for MC production with realistic conditions for Phase1 2024
     'phase1_2024_realistic'        : '131X_mcRun3_2024_realistic_v3',
     # GlobalTag for MC production with realistic conditions for Phase2
-    'phase2_realistic'             : '131X_mcRun4_realistic_v4'
+    'phase2_realistic'             : '131X_mcRun4_realistic_Candidate_2023_05_06_19_12_47'
```

Comment:
@cms-sw/alca-l2 if this PR is agreed upon, we'll need a full-fledged GloblaTag implementing this.

Reply:
Ciao @mmusich thanks for this!
In general it looks good, give me just a bit more of time to review the updated logic and we will provide a versioned GT based on your candidate.